frontend/src/_components/SearchBox.jsx

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import useDebounce from '@/_hooks/useDebounce';
 
-export function SearchBox({ width = '200px', onSubmit, debounceDelay = 300 }) {
+export function SearchBox({ width = '200px', onSubmit, debounceDelay = 300, darkMode = undefined }) {
```

Comment:
```suggestion
export function SearchBox({ width = '200px', onSubmit, debounceDelay = 300, darkMode = false }) {
```

Instead of undefined, we can have it as false